web/app/controllers/user_msg.php

```diff
@@ -141,6 +141,9 @@ function getHistory() {
 			height: calc(100ch - 10rem);
 		}
 	}
+	#conversations {
+		border-radius: var(--bs-border-radius) !important;
+	}
```

Comment:
Please add the `rounded` class to `#conversation` element instead of adding a CSS rule here.